cgo/lapack_bench_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cgo
+
+import (
+	"testing"
+
+	"github.com/gonum/lapack/testlapack"
+)
+
+func BenchmarkDbdsqr10x10(b *testing.B) {
```

Comment:
`gofmt` won't let the last function be a one liner for some reason.  The spacing was just a slip on my part.